docker-entrypoint.sh

```diff
@@ -3,13 +3,13 @@
 # set -e stops the execution of a script if a command or pipeline has an error
 set -e
 
-S3_MANAGER_URL="http://s3-manager-api.service.strato:7540/api/v2/object-stores"
+S3_VAULT_URL="http://127.0.0.1:8600/_/healthcheck"
 
-if [[ "$VERIFY_SERVICE_ENABLED" ]]; then
+if [[ "$VERIFY_S3_VAULT_ACCESSIBLE" ]]; then
```

Comment:
All this if is not needed anymore. It was my hack not to start scality before it's enabled by the user.

Reply:
see this commit: 1a148ec775667f115ab4909987c9e450a5f7eab8

Reply:
Why would you need scality managed service if there is no vault to authorize requests?